src/BenchmarksClient/Startup.cs

```diff
@@ -151,16 +151,11 @@ private static Process StartProcess(ClientJob job)
                 }
             }
 
-            if (job.PipelineDepth > 0)
-            {
-                command += $" -s scripts/pipeline.lua";
-            }
-
-            command += $" {job.ServerBenchmarkUri}";
+            command += $" -s scripts/pipeline.lua {job.ServerBenchmarkUri} -- {job.Method}";
```

Comment:
Once we have a separate script for non-GET methods, it should be used instead of `pipeline.lua` but only when `job.Method != "GET"`